app/elements/common/settings/change-password-dialog.html

```diff
@@ -26,6 +28,7 @@
 			on-response="_onResponse"
 			on-error="_onError">
 		</iron-ajax>
+		<match-password other-password="{{password.newPassword}}" validator-name="match-password-validator"></match-password>
```

Comment:
Could use `[[password.newPassword]]` as you are only binding in one direction and this likely is cheaper.